setup.cfg

```diff
@@ -16,6 +16,7 @@ classifiers =
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
```

Comment:
Shouldn't this be in #126?

Reply:
I've actually included py39 in tox.ini before so it picks it up...

Reply:
Ah, fair enough. If we can, I'd like to wait for #126. If it has problems materializing we'd have to remove this classifier.

Reply:
I don't think we need per se. Identified the issue and it's a virtualenv bug... Our test suite is created with virtualenv, hence showing up in the ci .. but as you think 👍not sure if we can limit the upper constraint here 🤔